src/components/menu/LoadModule.js

```diff
@@ -187,6 +191,30 @@ class LoadModule extends Component {
         </div>
         )
 
+    case 'local-storage':
+      return (
+        <div className="col-4 nopadding">
+          <ul className="LoadModule-list">
+            {Object.keys(this.state.localStorageModules || {}).map(moduleName => (
+              <li key={moduleName}>
+                <div className="btn-group">
+                  <button className='btn btn-link' onClick={() => {
+                    this.loadModule(JSON.stringify(this.state.localStorageModules[moduleName]))
+                  }}>
+                    {moduleName}
```

Comment:
One thought: how hard would it be to add an extra label with the date the file was last saved? If you have to restructure everything don't worry about it because it's not that important, but if it's easy to add I think it would be helpful to know what's safe to clean up later
![image](https://user-images.githubusercontent.com/13512036/126699422-427b4f4e-f8d6-48bc-9d76-205f9032dd4a.png)